### PR TITLE
DDF-2890 Added developing sections to templated docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds template update for last sections of jbake templated docs

#### Who is reviewing it? 

@mcalcote @vinamartin @emmberk @Lambeaux @jrnorth 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### What are the relevant tickets?
[DDF-2890](https://codice.atlassian.net/browse/DDF-2890)
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_developingComponents/developing-applications-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,174 @@
+:title: Developing Applications
+:type: developingComponemt
+:status: published
+:summary: Using OSGi within ${ddf-branding}.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_developingComponents/developing-applications-contents.adoc]: Is this summary correct?

Comment (ricklarsen) [reply distribution/docs/src/main/jdocs/content/_developingComponents/developing-applications-contents.adoc]: nope. fixed.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**